src/app/search/MainSearchBarActions.tsx

```diff
@@ -107,21 +107,21 @@ function MainSearchBarActions({
   }
 
   const bulkTag = loadingTracker.trackPromise(async (selectedTag: TagValue) => {
-    if (selectedTag === 'lock' || selectedTag === 'unlock') {
-      // Bulk locking/unlocking
-      const state = selectedTag === 'lock';
-      const lockables = filteredItems.filter((i) => i.lockable);
-      dispatch(bulkLockItems(lockables, state));
-    } else {
-      // Bulk tagging
-      const tagItems = filteredItems.filter((i) => i.taggable);
-
-      if (isTagValue(selectedTag)) {
-        dispatch(bulkTagItems(tagItems, selectedTag));
-      }
+    // Bulk tagging
+    const tagItems = filteredItems.filter((i) => i.taggable);
+
+    if (isTagValue(selectedTag)) {
+      dispatch(bulkTagItems(tagItems, selectedTag));
     }
   });
 
+  const bulkLock = loadingTracker.trackPromise(async (selectedTag: TagValue) => {
+    // Bulk locking/unlocking
+    const state = selectedTag === 'lock';
+    const lockables = filteredItems.filter((i) => i.lockable);
```

Comment:
I wonder if we should also filter on the is already locked/unlocked so we can skip some requests. Just would need to hope that DIM matches approximate game state.